source/features/hide-promoted-tweets.js

```diff
@@ -0,0 +1,3 @@
+export default function () {
+	$('.promoted-tweet').parent().remove();
```

Comment:
Use `.hide()`. We should modify the DOM as little as possible so not create an unexpected DOM for the Twitter JS.